scripts/fornax1.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+
+set -x
+
+export a=
+export b=
+export c=
+
+pushd /root
+hostnamectl set-hostname node-a
+ufw disable
+swapoff -a
+apt-get -y update
+echo -e 'br_netfilter' | cat > /etc/modules-load.d/k8s.conf
+echo -e 'net.bridge.bridge-nf-call-ip6tables = 1\nnet.bridge.bridge-nf-call-iptables = 1' | cat >> /etc/sysctl.d/k8s.conf
+sysctl --system
+apt-get -y update
+apt-get install docker.io -y
+systemctl enable docker
+systemctl start docker
+apt-get -y update
+apt-get install -y apt-transport-https ca-certificates curl
+curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+echo -e 'deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main' | cat >> /etc/apt/sources.list.d/kubernetes.list
+apt-get -y update
+apt-get install -qy kubelet=1.21.1-00 kubectl=1.21.1-00 kubeadm=1.21.1-00
+apt-mark hold kubelet kubeadm kubectl
+systemctl enable docker.service
+kubeadm init
+export KUBECONFIG=/etc/kubernetes/admin.conf
+export kubever=$(kubectl version | base64 | tr -d '\n')
+kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
+sleep 40s
+kubectl get nodes
+cp /etc/kubernetes/admin.conf $HOME/
+chown $(id -u):$(id -g) $HOME/admin.conf
+export KUBECONFIG=$HOME/admin.conf
+echo y | sudo apt-get install vim
+GOLANG_VERSION=${GOLANG_VERSION:-"1.14.15"}
+apt -y install make gcc jq
+wget https://dl.google.com/go/go1.14.15.linux-amd64.tar.gz -P /tmp
+tar -C /usr/local -xzf /tmp/go1.14.15.linux-amd64.tar.gz
+echo -e 'export PATH=$PATH:/usr/local/go/bin\nexport GOPATH=/usr/local/go/bin\nexport KUBECONFIG=/etc/kubernetes/admin.conf' |cat >> ~/.bashrc 
+sleep 10s
+cp /usr/local/go/bin/go  /usr/local/bin
+source /root/.bashrc
+go version
```

Comment:
commands like this were used in manual testing but are they useful here in a script?